pkg/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -322,6 +322,33 @@ func TestHasMachineSetOwner(t *testing.T) {
 
 }
 
+func TestHasMachineRemediationDisabled(t *testing.T) {
+	machineWithRemediationDisabled := maotesting.NewMachine("machineWithRemediationDisabled", "node")
```

Comment:
`maotesting` isn't defined, see https://travis-ci.com/kubevirt/machine-remediation-operator/builds/121625308#L916
You'll need to look for other calls to `NewMachine` to see what is appropriate for this repo